scale/scale.go

```diff
@@ -0,0 +1,90 @@
+package scale
+
+import (
+	"github.com/pkg/errors"
+
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/client/restclient"
+	kclient "k8s.io/kubernetes/pkg/client/unversioned"
+)
+
+// for testing purposes
```

Comment:
unnecessary comment. also, if you did want to have a comment on a type (which is always nice), it should be a godoc-appropriate comment.